scripts/write_complete_language_data.py

```diff
@@ -0,0 +1,168 @@
+        for valid_word in valid_words:
+            power_of_10_num = pow(10, power_of_10)
+            if power_of_10 >= 2:
+                language_data["MULTIPLIERS"][valid_word] = power_of_10_num
+        return
+    valid_word = ""
+    if "[" in required_part:
+        valid_word = required_part.split("[")[0].strip()
+    else:
+        valid_word = required_part.split(">")[0].strip()
+
+    power_of_10_num = pow(10, power_of_10)
+    if power_of_10 >= 2:
+        language_data["MULTIPLIERS"][valid_word] = power_of_10_num
+
+
+def _extract_information(key, word, language_data):
+    """Identify the type of number - simple,compound,multiplier."""
+    try:
+        number = int(key)
+    except ValueError:
+        print("The given key {} is not an integer".format(key))
+        return
+
+    word = word.replace(";", '')
+    count_greater_than_sign = word.count('>')
+    count_less_than_sign = word.count('<')
+    count_equal_to_sign = word.count('=')
+    if count_equal_to_sign != 0:
+        return
+    if count_greater_than_sign == 0 and count_less_than_sign == 0:
+        _add_base_words(number, word, language_data)
+    elif count_greater_than_sign == 2 and count_less_than_sign == 0:
+        _add_compound_words(number, word, language_data)
+    elif count_greater_than_sign == 2 and count_less_than_sign == 2:
+        _add_multiplier_words(number, word, language_data)
+
+
+def write_complete_data():
+    """
+    Main function to extract data from source files , merge with supplementary data
+    and write the combined results to the final target directory.
+    """
+    for file_name in os.listdir(SOURCE_PATH):
+        full_source_path = os.path.join(SOURCE_PATH, file_name)
+        full_target_path = os.path.join(TARGET_PATH, file_name.split(".")[0]+".py")
+        full_supplementary_path = os.path.join(SUPPLEMENTARY_PATH, file_name)
+
+        language_data = {}
+        ordered_language_data = OrderedDict()
+        with open(full_source_path, 'r') as source:
+            for keys in REQUIRED_DATA_POINTS:
+                language_data[keys] = {}
+                ordered_language_data[keys] = {}
+
+            data = json.load(source)
+            try:
+                requisite_data = data['rbnf']['rbnf']['SpelloutRules']
+            except KeyError:
+                print("This key doesn't exist in {}".format(file_name))
+                continue
+
+            for keys, vals in requisite_data.items():
+                if _is_valid(keys):
+                    for key, val in vals.items():
+                        _extract_information(key, val, language_data)
+
+        with open(full_supplementary_path, 'r') as supplementary_data:
+            data = json.load(supplementary_data)
+            for keys in REQUIRED_DATA_POINTS:
+                language_data[keys].update(data[keys])
+                sorted_tuples = sorted(language_data[keys].items(), key=lambda x: x[1])    
```

Comment:
What about sorting by value and _then_ by word?

```suggestion
                sorted_tuples = sorted(language_data[keys].items(), key=lambda x: (x[1], x[0]))    
```